_data/Kuwarbir/Kuwarbir.md

```diff
@@ -0,0 +1,8 @@
+---
+name: Kuwarbir Singh
+institution: Chitkara University,Punjab,India
+profile_pic: kuwarbir.png 
+quote: "Thank you all my teachers, fellow students who have been a constant support to me,
```

Comment:
Quote must **not** be longer than 100 characters, yours is **217** characters.

Reply:
Made changes in quote